scripts/shell/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pc-ble-driver-py>=0.15.0
```

Comment:
i've tested only on v.0.14.2 so can't be sure if this works from my perspective
if scripts works with v0.15.0 ignore this comment :)

Reply:
i just added what i tested with.